docs/docs/whether.md

```diff
@@ -0,0 +1,104 @@
+# Should you use JSpecify annotations in your Java code?
+
+With the release of JSpecify 1.0, we guarantee backwards compatibility: we will
+not rename the annotations or move them or make other changes that would cause
+your compilation to fail when you update.
+
+However, there are other things to think about when deciding whether to start
+using JSpecify annotations in your code, including at least:
+
+*   If your Java code doesn’t use nullness annotations already, it should!
+*   How well does the current version of your nullness checker support JSpecify
+    annotations?
+*   Do you have Kotlin users? Which compiler version do they use?
+*   Do you use whole-program annotation processors, such as [Dagger]?
+
+## If your Java code doesn’t use nullness annotations yet
+
+If your Java code doesn’t already use nullness annotations, we recommend that
+you [start using JSpecify annotations](using).
+
+Even if you are not currently using a nullness analyzer, applying JSpecify
+annotations can still provide benefits. * The annotations are useful
```

Comment:
This list got turned into a paragraph with asterisks in it. If the paragraphing was intentional, then remove the asterisks; if not, re-listify it, and consider removing the transition words at the start of the later sentences/bullets.